packages/code-du-travail-frontend/src/conventions/Convention/Info.js

```diff
@@ -18,34 +27,60 @@ const Info = ({ container: { idcc, categorisation, date_publi, url } }) => {
               <td>{format(parseISO(date_publi), "dd/MM/yyyy")}</td>
             </tr>
           )}
-          {categorisation && (
-            <tr>
-              <th>Categories</th>
-              <td>
-                {categorisation.map((categorie, index) => (
-                  <div key={index}>{categorie}</div>
-                ))}
-              </td>
-            </tr>
-          )}
+          <tr>
+            <th>Nombre de textes</th>
+            <td>{nbTextes}</td>
+          </tr>
+          <tr>
+            <th>Nombre d’articles</th>
+            <td>
+              {nbArticles.vigueurEtendu + nbArticles.vigueurNonEtendu}
```

Comment:
A mon avis la pluralize fonction est un peu "too smart for her own good" et trop spécifique pour être vraiment générisée dans `lib` encore (le cas ou on a besoin du chiffre + des pluriels sont inexistants dans le proj à part ici). Mais je suis d'accord qu'une fonction non générique (qui resterait dans ce fichier) qui permette d'avoir des noms de variable plus courts (dans l'idée de pluralize avec `nb`) pour faciliter la lecture et la compréhension de la logique pour générer le wording ici ferait du bien !

Reply:
pour l'utilité de pluralize , je pense qu'en cherchant un petit peu il y a plusieurs endroit qui pourrait en bénéficier comme par exemple marque Source / Sources en fonction du nombre d'article cité dans les pages Résultats des simulateurs.